src/models/behaviors.js

```diff
@@ -397,19 +547,94 @@ function copy (originalRequest, responsePromise, copyArray, logger) {
         copyArray.forEach(function (copyConfig) {
             var from = getFrom(originalRequest, copyConfig.from),
                 using = copyConfig.using || {},
-                fnMap = { regex: regexValue, xpath: xpathValue, jsonpath: jsonpathValue },
+                fnMap = {
+                    regex: regexValue,
+                    xpath: xpathValue,
+                    jsonpath: jsonpathValue
+                },
                 values = [];
 
             if (fnMap[using.method]) {
                 values = fnMap[using.method](from, copyConfig, logger);
             }
-
             replace(response, copyConfig.into, values, logger);
         });
         return Q(response);
     });
 }
 
+function CSVData (CSVPath, ColumnMatch, result, values, index) {
+    var flag = true;
```

Comment:
Can you use a better name for this? Glancing through the code, it appears to have something to do with the key being found, right? If so, maybe name it `keyFound` to give some hint as to what it's doing.